docker-compose.yml

```diff
@@ -24,6 +24,8 @@ services:
   # Django web application
   django: &django
     <<: *global # this will inherit all the envs from x-global-environment
+    ports:
+      - 8003:8003
```

Comment:
similarly not required as everything is proxied through caddy but it was easier to debug issues hitting the container directly